### PR TITLE
As unselectable features are not displayed in gray correcly on Ubuntu, show them additionally in italics

### PR DESCRIPTION
**Pull request by ekuiter**
See the discussion in issue #1055.

### CODE REVIEW

File: plugins/de.ovgu.featureide.fm.core/library/subant_build_jar.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (SundermannC) [inline plugins/de.ovgu.featureide.fm.core/library/subant_build_jar.xml:1]: Are these changes intended? How are they related to the issue?

**Review by SundermannC — COMMENTED** [review]
The fix for the issue looks good. However, there are also changes to an ant file. Is this intentional? How is this related to the issue?

File: plugins/de.ovgu.featureide.fm.core/library/subant_build_jar.xml (hunk 1 above)
Comment (ekuiter) [reply plugins/de.ovgu.featureide.fm.core/library/subant_build_jar.xml:1]: This seems to have happened due to cafab33 from Sebastian. I don't know why it's in here, it is related to #1100.

**Comment by ekuiter** [comment]
I created a new pull request #1165 without these changes to the XML file.
